jarbas/api/views.py

```diff
@@ -30,6 +30,11 @@ def get(self, request):
         if self._bool_param('suspicions'):
             self.queryset = self.queryset.suspicions()
 
+        # filter has_receipt_url
+        has_receipt_url = self._bool_param('has_receipt_url')
+        if has_receipt_url is not None:
+            self.queryset = self.queryset.has_receipt_url(has_receipt_url)
+
```

Comment:
I was thinking… If we're offering this filter for the JSON API we need to document that in the `README.md` and write tests for that. You can either update the documentation and add the tests, or stash this change for another PR focusing on the API.

Reply:
I think I'll add it to this PR, since it's quite a small change. What you think?

Reply:
That'd be great ; ) Tks!